json_serializable/lib/src/type_helpers/enum_helper.dart

```diff
@@ -34,6 +34,11 @@ class EnumHelper extends TypeHelper {
     var wrappedExpression =
         simpleExpression.hasMatch(expression) ? expression : '{$expression}';
 
+    var closureArg = consts.closureArg;
```

Comment:
so was the import unused before? How is it that you could add a prefix without changing any other lines?

Reply:
It wasn't unused before. I'm creating a local var that maps exactly to the field `closureArg`. This way the code that does the generation doesn't change at all...